pkgs/applications/science/biology/neuron/default.nix

```diff
@@ -1,94 +1,103 @@
 { lib, stdenv
 , fetchurl
-, pkg-config
-, automake
-, autoconf
-, libtool
-, ncurses
 , readline
-, which
-, python ? null
-, useMpi ? false
+, xorg
 , mpi
-, iv
+, cmake
+, bison
+, flex
+, git
+, perl
+, gsl
+, xcbuild
+, python3
+, usempi ? false
+, useiv ? true
+, core ? false
+, rx3d ? false
 }:
 
 stdenv.mkDerivation rec {
-  pname = "neuron${lib.optionalString useMpi "-mpi"}";
-  version = "7.5";
-
-  nativeBuildInputs = [ which pkg-config automake autoconf libtool ];
-  buildInputs = [ ncurses readline python iv ]
-    ++ lib.optional useMpi mpi;
-
+  pname = "neuron";
+  version = "8.2.1";
+  # format is for pythonModule conversion
+  format = "other";
+  propagatedBuildInputs = [
```

Comment:
I am still very skeptical if this must be propagated

Reply:
I have now tested and adjusted, thank you for pointing it out.